src/network/serverpackethandler.cpp

```diff
@@ -969,8 +969,9 @@ bool Server::checkInteractDistance(RemotePlayer *player, const f32 d, const std:
 	else if (max_d < 0)
 		max_d = BS * 4.0f;
 
-	// cube diagonal: sqrt(3) = 1.732
-	if (d > max_d * 1.732) {
+	// Cube diagonal * 1.5 for maximal supported node extents:
+	// sqrt(3) * 1.5 ≅ 2.6
+	if (d > max_d + 2.6f * BS) {
```

Comment:
And the `* 1.732`? Why is it gone?

Reply:
2.6 =~= 1.732 * 1.5

Reply:
The `* 1.732` was there to account for a block diagonally `d` nodes away, making the distance `sqrt(3) * d`. The `+ 2.6` quite obviously won't be able to cover this.

Here's a (terrible) drawing illustrating the problem in the 2D space:
![unbenannt](https://user-images.githubusercontent.com/1042418/50724048-270a7f80-10e7-11e9-9489-cfcd00edb7a0.png)

Reply:
After some testing it seems this isn't actually a problem in practice, though I'm sure it was added for this purpose.

Reply:
![Demo](https://user-images.githubusercontent.com/1497498/50724304-92565080-10eb-11e9-8a79-a3b8c9ca7625.png)